api/src/wfl/service/firecloud.clj

```diff
@@ -134,28 +134,26 @@
   [workspace submission-id]
   (get-workspace-json workspace "submissions" submission-id))
 
+;; Further work needed if we needed to specify multiple `include-key`s.
+;;
 (defn get-workflow
   "Query the `firecloud-url` for the the `workflow` created by the `submission`
-   in the Terra `workspace`."
-  [workspace submission-id workflow-id]
-  (get-workspace-json workspace "submissions" submission-id "workflows" workflow-id))
+   in the Terra `workspace`, optionally restricting output to `include-key`."
+  ([workspace submission-id workflow-id include-key]
+   (let [parts [workspace "submissions" submission-id "workflows" workflow-id]]
+     (-> (apply workspace-api-url parts)
+         (http/get {:headers      (auth/get-auth-header)
+                    :query-params (when include-key {:includeKey include-key})})
```

Comment:
It takes significantly more time to fetch the entire workflow vs. when we restrict the returned keys to only those we need.  Batch team suggested that when polling active workflows in Firecloud for the sole purpose of updating their status, we can use the `includeKey` parameter to only return the status.

I did some informal runtime analysis:
```
(let [workspace "pathogen-genomic-surveillance/CDC_Viral_Sequencing_dev"
        submission-id "d33b55de-e309-4463-a56e-439dab0c344e"
        workflow-id "c484734c-28d8-47df-8cdc-278bfa5e4e21"
        include-key "status"]
    (log/info "START fetch workflow all keys")
    (get-workflow workspace submission-id workflow-id)
    (log/info "END fetch workflow all keys")
    (log/info "START fetch workflow status only")
    (get-workflow workspace submission-id workflow-id include-key)
    (log/info "END fetch workflow status only"))
{"severity":"INFO","logging.googleapis.com/sourceLocation":{"file":"/Users/okotsopo/wfl/api/src/wfl/service/firecloud.clj","line":5},"timestamp":"2022-01-12T19:48:16.480991Z","message":"START fetch workflow all keys"}
{"severity":"INFO","logging.googleapis.com/sourceLocation":{"file":"/Users/okotsopo/wfl/api/src/wfl/service/firecloud.clj","line":7},"timestamp":"2022-01-12T19:48:39.300331Z","message":"END fetch workflow all keys"}
{"severity":"INFO","logging.googleapis.com/sourceLocation":{"file":"/Users/okotsopo/wfl/api/src/wfl/service/firecloud.clj","line":8},"timestamp":"2022-01-12T19:48:39.300707Z","message":"START fetch workflow status only"}
{"severity":"INFO","logging.googleapis.com/sourceLocation":{"file":"/Users/okotsopo/wfl/api/src/wfl/service/firecloud.clj","line":10},"timestamp":"2022-01-12T19:48:44.757706Z","message":"END fetch workflow status only"}
```

Fetching the whole workflow: 26s
Fetching just the workflow status: 5s